commands/command_track.go

```diff
@@ -181,16 +184,24 @@ func findAttributeFiles() []string {
 		paths = append(paths, repoAttributes)
 	}
 
-	filepath.Walk(config.LocalWorkingDir, func(path string, info os.FileInfo, err error) error {
-		if err != nil {
-			return err
+	fchan, errchan := tools.FastWalkGitRepo(config.LocalWorkingDir)
```

Comment:
It's only actually used once in production code, the other place is in the tests. Since what this `findAttributeFiles` function is doing is only hooking out items from the channel that are `.gitattributes` files it seems to me to make sense like it is, processing the channel and skipping anything else. I certainly don't want another function that builds a full list and then needs to be iterated, the other option is a function which takes search criteria but that's pretty specific and nothing else is using it yet. Maybe if something else needs it later it could be refactored but wrapping it speculatively right now might not actually be re-usable anyway.